as pld sets CMAKE_BUILD_TYPE to PLD, conditions like this fail:

if (${CMAKE_BUILD_TYPE} MATCHES "Release")

mabe PLDRelease would do better?

--- hhvm-HHVM-3.2.0/CMake/HPHPSetup.cmake~	2014-07-18 03:12:10.000000000 +0300
+++ hhvm-HHVM-3.2.0/CMake/HPHPSetup.cmake	2014-08-06 11:57:50.109380674 +0300
@@ -81,7 +81,7 @@
   add_definitions(-D_GNU_SOURCE)
 endif()
 
-if(${CMAKE_BUILD_TYPE} MATCHES "Release")
+if(NOT ${CMAKE_BUILD_TYPE} MATCHES "Debug")
   add_definitions(-DRELEASE=1)
   add_definitions(-DNDEBUG)
   message("Generating Release build")
